### PR TITLE
Updating Issue template

### PR DESCRIPTION
**Pull request by Dinesh101041**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+git rev-parse HEAD; git diff
+
+<put code or diff here>
+```
+2. What exact command you run:
+3. __Full logs__ or other relevant observations:
+```
+<put logs here>
+```
+4. please simplify the steps as much as possible so they do not require additional resources to run,
```
Comment (steven-esser) [inline .github/ISSUE_TEMPLATE/bug-.md]: This is not really relevant to an Electron GUI application.

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-.md (hunk 1 above)
Comment (Dinesh101041) [reply .github/ISSUE_TEMPLATE/bug-.md]: Changed sir please check

**Review by steven-esser — CHANGES_REQUESTED** [review]
@Dinesh101041 Thank you. Can you please sign off your commits and rebase them into a single, signed commit?

**Comment by Dinesh101041** [comment]
Sir can your explain more what to do  because  i am not well in DCO workflow and i make the pull request through github web ui so unable to run a force command 
or can i delete the pull request and branch and make a new pull request sir

**Comment by pratik0316** [comment]
Hi @Dinesh101041 , first rebase all commits to a single commit then , just put the flag `-s` when you are making the commit, to correct DCO .
Its is better to use the CLI always
You can follow this https://www.internalpointers.com/post/squash-commits-into-one-git

**Comment by Dinesh101041** [comment]
Thank you sir i will do it 
in between i ran into another doubt when setting scancode-workbench locally using node js
i cloned the repo and done the `npm install` command 
after that how to run a 'Rebuild native Node.js modules against the app version of Node.js'  for windows 
i run this command ` .\node_modules\.bin\electron-rebuild.cmd`but it shows error
and after the running  above command and npm start command we can upload a file into application?
 i am asking because of first time i run the `npm run ` command without  running rebulid native command  the application is launched but i am unable to upload files 

can you help me out sir

**Comment by pratik0316** [comment]
I didn't get u correctly , u are not able to upload in git?

**Comment by Dinesh101041** [comment]
no sir after running npm start command 
the application is launched 
image

but after clicking `import json` or `open sql` buttons i did not get any response sir

**Comment by pratik0316** [comment]
can you post the console logs if any?

**Comment by Dinesh101041** [comment]
Sorry for late reply sir here is console log 
image

and when running rebulid command getting exit code error sir
image

**Comment by pratik0316** [comment]
@Dinesh101041 ki think it's due to py version mismatch , deactivate the anaconda environment and give a try once

**Comment by Dinesh101041** [comment]
done a rebase sir

**Comment by pratik0316** [comment]
@Dinesh101041 okay , just signoff commits

**Comment by Dinesh101041** [comment]
please check now sir

**Comment by Aayush-hub** [comment]
@Dinesh101041 You DCO check failure suggests that you didn't sign off the commits. Please run followings commands :
- `git rebase HEAD~9 --signoff`
- `git push --force-with-lease origin develop`

**Comment by Dinesh101041** [comment]
i am getting merging conflicts on signof commit sir i am closing this pull request and create new one with all updation

**Pull request closed without merging**